src/content/docs/journey-large-logs/filter.mdx

```diff
@@ -0,0 +1,143 @@
+  src={logsIngestPipeline}
+/>
+
+<figcaption>
+  During the ingestion process, customer log data can be parsed, transformed, or dropped before being stored in the New Relic database (NRDB).
+</figcaption>
+
+## Filter your log ingest [#filter-steps]
+
+The following steps will guide you through how to drop logs in the New Relic UI.
+
+
+Let's say Acme Corp creates 2TB of logs each day. They decide this too many logs to ingest for both cost and usability reasons. They take a look at their logs and realize over half of their daily logs are from a legacy Node.js application. While they need to keep the app around, they don't care for the logs created by this app. They decided to drop all logs created from the Node app.
+
+
+<Steps>
+    <Step>
+        ### Navigate to the UI
+
+        Go to **[one.newrelic.com > Logs](https://one.newrelic.com/launcher/logger.log-launcher)**
+    </Step>
+    <Step>
+        ### Create your drop rule
+
+        Filter or query to the specific set of logs that contain the data to be dropped.
+
+        There are a few ways to do this, but the easiest is to query for the logs you want to drop. In this case, ACME Corp would do the following:
+
+
+<SideBySide>
+    <Side>
+
+        1. Select **All partitions** near the search bar
+        2. Enter their query. In this case `logtype=node` 
+        3. Press enter and confirm the correct logs appear. 
+        4. Once the query is active, from **Manage data** on the left nav of the Logs UI, click **Create drop filter**.
+        5. Give the drop rule a meaningful name.
+        6. Save the drop filter rule.
+    </Side>
+    <Side>
+        <img
+        title="Query log "
+        alt="An image showing how to query for the logs you will drop"
+        src={logsDrop}
+         />
+    </Side>
+</SideBySide>
+
+    </Step>
+
+    <Step>
+        ### Drop attributes
+
+        ACME Corp still wants to reduce their ingest. They decided that they don't need certain attributes in their stored logs, so they decide to drop attributes such as `purchase_order`.
+
+<SideBySide>
+    <Side>
+
+        1. In **All logs** click on a log that contains the attribute you want to drop to open log detail view.
+        2. Click on the attribute you want to drop to open the attribute menu. In this case `purchase_order`.
+        3. Click **Create drop filter from attribute**.
+        4. Give the drop rule a meaningful name.
+        5. Save the drop filter rule.    
+    </Side>
+    <Side>
+        <img
+        title="Query attribute"
+        alt="An image showing how to query for the attributes you will drop"
+        src={logsAttribute}
+         />
+    </Side>
+</SideBySide>
+    </Step>
+</Steps>
+
+Repeate the above steps as many times as required until you're happy with your log ingest. If you need help querying for logs and attributes, [check out our doc on log specific syntax](/docs/logs/ui-data/query-syntax-logs/).
```

Comment:
we don't normally link other docs from tutorials, but here i think we should link to the deeper-dive feature docs on log filtering.